pkg/sinks/forward/forward.go

```diff
@@ -111,9 +111,9 @@ func NewDataForward(
 }
 
 // Start starts reading the buffer and forwards to sinker. Call `Stop` to stop.
-func (df *DataForward) Start() <-chan struct{} {
+func (df *DataForward) Start() <-chan error {
 	log := logging.FromContext(df.ctx)
-	stopped := make(chan struct{})
+	stopped := make(chan error)
```

Comment:
Should we introduce another channel dedicated for error?

Reply:
We had a offline discussion, we decided to go with a blocking call implementation for start which returns an error similar to http [server](https://pkg.go.dev/net/http#ListenAndServeTLS). We will make this change after 1.4.